src/psm/README.md

```diff
@@ -10,72 +11,138 @@ Features:
-- ``dx,dy``: (optional) these arguments set the bump pitch to decide the voltage source location in the absence of a vsrc file. Default bump pitch of 140um used in absence of these arguments and vsrc.
-- ``net``: (mandatory) is the name of the net to analyze, power or ground net name.
-- ``enable_em``: (optional) is the flag to report current per power grid segment.
-- ``outfile``: (optional) filename specified per-instance voltage written into file.
-- ``error_file``: (optional) filename to write each connectivity error into.
-- ``em_outfile``: (optional) filename to write out the per segment current values into a file, can be specified only if enable_em is flag exists.
-- ``voltage``: (optional) Sets the voltage on a specific net. If this command is not run, the voltage value is obtained from operating conditions in the liberty.
-- ``node_density``: (optional)  This value can be specfied by the user in um to determine the node density on the std. cell rails. Cannot be used together with node_density_factor.
-- ``node_density_factor``: (optional) Integer value factor which is multiplied by standard cell height to determine the node density on the std. cell rails. Cannot be used together with node_density. Default value is 5.
-- ``corner``: (optional) Corner to use for analysis.
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-net` | Name of the net to analyze, power or ground net name. |
+
+### Write Spice Power Grid
+
+```tcl
+write_pg_spice
+    [-vsrc vsrc_file]
+    [-outfile out_file]
+    [-net net_name]
+    [-dx bump_pitch_x]
+    [-dy bump_pitch_y]
+    [-corner corner]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-vsrc` | File to set the location of the power C4 bumps/IO pins. [Vsrc_aes.loc file](test/Vsrc_aes_vdd.loc) for an example with a description specified [here](doc/Vsrc_description.md). |
+| `-dx`,`-dy` | These arguments set the bump pitch to decide the voltage source location in the absence of a vsrc file. Default bump pitch of 140um used in absence of these arguments and vsrc. |
+| `-net` | Name of the net to analyze, power or ground net name. |
+| `-outfile` | Filename specified per-instance voltage written into file. |
+| `-corner` | Corner to use for analysis. | 
+
+### Set PDNSim Net voltage
+
+```tcl
+set_pdnsim_net_voltage
+    [-net net_name]
+    [-voltage volt]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-net` | Name of the net to analyze, power or ground net name. |
+| `-voltage` | Sets the voltage on a specific net. If this command is not run, the voltage value is obtained from operating conditions in the Liberty file. |
+
+### Useful developer functions
+
+If you are a developer, you might find these useful. More details can be found in the [source file](./src/pdnsim.cpp) or the [swig file](./src/pdnsim.i).
+
+```tcl
+# GUI debugging 
+debug_gui
+```
```

Comment:
Where is this command?

Reply:
Seems it has been removed. Reflected in latest commit